arduino/WandCore/CAPClient.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+  Capacitive Touch Client
+
+  Manages a Touch Surface Input
+*/
+
+#include "CAPClient.h"
+
+// touch threshold
+const int touchThold = 50; //above means not touched
+
+// Touch surface timing variables
+int debounce = 20;                // ms debounce period to prevent flickering when pressing or releasing the button
+int DCgap = 250;                  // max ms between clicks for a double click event

```

Comment:
IDK about you, but I don't really like the `DC*` nomenclature, a the very least make it `DT` since "click" doesn't make sense any more, but I'd prefer `doubleClickGap` (I generally prefer detail in variable names). The comment could be updated, too.